test/e2e/tests/confirmations/helpers.ts

```diff
@@ -151,3 +151,27 @@ export async function mockSignatureRejectedWithDecoding(
 export async function mockPermitDecoding(mockServer: Mockttp) {
   return [await createMockSignatureDecodingEvent(mockServer)];
 }
+
+export async function mockedSourcifyTokenSend(mockServer: Mockttp) {
```

Comment:
this has been moved from the spec file to helpers file, to avoid the run of all the specs, whenever this is imported in another spec file